src/readerModel.ts

```diff
@@ -98,6 +98,18 @@ export default class ReaderModel {
     return this.modelFor(models.Isp, 'ISP', ipAddress, 'isp()');
   }
 
+  /*
+   * Returns the Enterprise db data for an IP address
+   *
+   * @param ipAddress The IP Address you want to query the Enterprise db with
+   *
+   * @throws {BadMethodCallError} Throws an error when the DB doesn't support Enterprise queries
+   * @throws {AddressNotFoundError} Throws an error when the IP address isn't found in the database
+   */
+  public enterprise(ipAddress: string): models.City {
+    return this.modelFor(models.City, 'Enterprise', ipAddress, 'enterprise()');
```

Comment:
I think we should create a `models/Enterprise.ts` that is an `Enterprise` class that just extends the `City` model. The difference between the code above and the suggestion is subtle, but imagine the following scenario:

```
const response = reader.enterprise('128.101.101.101');
console.log(response.constructor) // 'City'
```

In the code above, it'd be reasonable for `Enterprise` to be returned in the console log.

Defining an `Enterprise` model class could also probably benefit some TypeScript usage.

Reply:
Sounds good.  Added in https://github.com/maxmind/GeoIP2-node/pull/8/commits/a769a95a58535e10070173279386067b1bcefa87